### ISSUE TITLE
unable to remove filename containing double-quote

### ISSUE THREAD

**Issue by IgnorantGuru** [IgnorantGuru/rmdupe#2]
Received elsewhere:  A report that rmdupe is unable to remove a file if the filename contains a double quote (").

**Comment by danilobellini** [comment]
When trying to remove a file named `Don`t Go`, the backslash troubled it as well.

```
/usr/local/bin/rmdupe: eval: line 211: unexpected EOF while looking for matching ``'
/usr/local/bin/rmdupe: eval: line 212: syntax error: unexpected end of file
```

**Comment by IgnorantGuru** [comment]
Thanks - these filename issues should now be corrected in 1.0.4.
